### ISSUE TITLE
Update documentation 

### ISSUE THREAD

**Issue by hschmidt** [custom-components/sensor.airthings_wave#43]
As a new Home Assistant user it wasn't immediately clear to me how to add multiple Airthings devices as sensors with a fixed MAC address. It might makes sense to include an example in the documentation:

```
sensor:
  - platform: airthings_wave
    scan_interval: 120
    elevation: 998
    mac: 00:81:F9:XX:XX:XX
  - platform: airthings_wave
    scan_interval: 120
    elevation: 998
    mac: 04:ee:03:YY:YY:YY
```

I also had difficulties to add even one device because it was previously registered with a Airthings Hub. What made it work was to 

1) Remove all sensors from the Airthings app for my phone
2) Turn off the Hub
3) Log into the Airthings web site
4) Unpair the Hub
5) Reset the Wave sensors by removing the batteries for at least 30 min as mentioned on https://help.airthings.com/en/articles/3119849-wave-basic-troubleshooting
6) Put the batteries back
7) Add the sensors in the Airthings app on the phone (this time without going through the hub)
8) Restart HomeAssistant

**Comment by MartyTremblay** [comment]
Thanks for this @hschmidt 

I don't have 2 devices to test with but I assume that these steps resolves #33 ?

**Comment by hschmidt** [comment]
Yes, unless the OP of #33 means auto-discovery.

**Comment by MartyTremblay** [comment]
There's a good chance that this issue has now been resolved as part of v3.0 release. pygatt has been replaced by bluepy which hopefully helps address this issue. 

Kindly test it and open an issue if this, or any other issues are discovered.

**Comment by hschmidt** [comment]
I think the wires got crossed here. I opened this issue to propose that the documentation be updated to explain 1) how to add more than one device by MAC address and 2) how to unpair devices from Airthings Hub because I have observed an existing pairing to prevent this integration from connecting to Airthings devices.

@MartyTremblay if you don't intend to update the docs, that's fine with me, but maybe you want to clearly state that. Your closing comment suggests that something was fixed in relation to this but as far as I can tell no documentation was updated.
